### DISCUSSION TITLE
Not sure where to add WebXR related packages

### DISCUSSION THREAD

**Discussion by De-Panther** [starikcetin/awesome-upm#47] [Suggestions]
I would like to add:
https://github.com/De-Panther/unity-webxr-export
WebXR Export - https://openupm.com/packages/com.de-panther.webxr/
WebXR Interactions - https://openupm.com/packages/com.de-panther.webxr-interactions/
https://github.com/De-Panther/webxr-input-profiles-loader
WebXR Input Profiles Loader - https://openupm.com/packages/com.de-panther.webxr-input-profiles-loader/

I'm not sure if they should be in Code, Framework or Native Plugins.

What do you think?

Thanks

**Comment by starikcetin** [comment]
Lines are blurry, but these are the general guidelines:

**Code:** Code libraries that are self-contained and flexible. You can use them wherever you want. 
For example: a simple math library

**Framework:** Code libraries that shape the way you develop your application, or ones that have major side-effects.
For example: Dependency Injection, ECS

**Native Plugins:** Code libraries that implement something native to the platform, but is not big enough or doesn't have enough major side-effects to be considered a framework. 
For example: a library that provides a way to use the device camera

I took a quick look, but can't find any code examples. If the library doesn't constrain the way people code (i.e. they can code the same way they would if your library was not there), then it should fall under Native Plugins. If the library requires a specific way of coding, or changes the way one usually structures the code, then it should fall under Frameworks category.

---

Now I think about it, maybe we should have a separate AR & VR category.

**Reply by De-Panther** [reply]
So should I add a PR with AR/VR category?

**Reply by starikcetin** [reply]
Yes let's do that actually

**Reply by De-Panther** [reply]
AR/VR category on top or VR/AR category at the bottom? 😁

**Reply by starikcetin** [reply]
Let's call it XR and put it at the bottom :)

**Reply by De-Panther** [reply]
Added PR
https://github.com/starikcetin/awesome-upm/pull/48
Thanks
